msteams-platform/apps-in-teams-meetings/teams-live-share-capabilities.md

```diff
@@ -506,30 +509,26 @@ timer.play();
 
 ---
 
-## Role verification for ephemeral data structures
+## Role verification for live data structures
 
-Meetings in Teams can range from one-on-one calls to all-hands meetings, and may include members across organizations. Ephemeral objects are designed to support role verification, allowing you to define the roles that are allowed to send messages for each individual ephemeral object. For example, you could choose that only meeting presenters and organizers can control video playback, but still allow guests and attendees to request videos to watch next.
+Meetings in Teams can range from one-on-one calls to all-hands meetings, and may include members across organizations. Live objects are designed to support role verification, allowing you to define the roles that are allowed to send messages for each individual live object. For example, you could choose that only meeting presenters and organizers can control video playback, but still allow guests and attendees to request videos to watch next.
```

Comment:
Had another thought, should we call out any limitations that role verification has. e.g. That we only check the user's roles every 5 minutes, and that in that time if an organiser changes the role of a participant from presenter to attendee, the role verifier will still return that they are a presenter

Reply:
Also not sure what happens in the SDK during this time too, will a newly attendee be still able to send presenter-limited events for up to 5 minutes?

Reply:
Talking to Pradeep, that yes, my assumption that the now-attendee will be able to send presenter-limited events

Reply:
We will update the docs in the next wave. Making a list, thank you for reviewing.